### PR TITLE
Update journal and source-week-5.md

### PR DESCRIPTION
**Pull request by rosalsm**

### CODE REVIEW

**Comment by karjac14** [comment]
Great Journal Rosa, you  covered a lot of  things we learned last week :+1:

**Comment by s4dmw** [comment]
looking good. :+1:

**Comment by ericquayson** [comment]
looks good :+1:  i like flukeout too!

File: _drafts/tutorial-week-5.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 My tutorial this week it will be about CSS selectors
```
Comment (kellymurray) [inline _drafts/tutorial-week-5.md:1]: I'd love to see a more instruction-based tutorial next week.

File: _drafts/journal-week-5.md [hunk 2]
```diff
@@ -27,3 +27,12 @@ DELETE. Removing data from the API
 HEAD. Getting just http header info.
 
 
+Last week wee learnt that an API endpoint is any URL that an API understands.
+The DOM is an API and a software is what we process in programming since the most simple calculator to the biggest complex code.
+
+We touched some concepts such as promises in jQuery.ajax()
+
+Last week we talked about minification also. Magnification is the fact of removing all of unnecessary spaces and carriage returns. As a result we have smaller files and faster downloads however we have to use regular spaces to read those files properly at human eye so before we read them, we translate them to regular files. Websites using jQuery usually use the min.js version to reduce download speeds.
```
Comment (kellymurray) [inline _drafts/journal-week-5.md:35]: Little bit of confusion - minification, not magnification! Easy mistake in English. :)

**Pull request merged**